src/test/java/com/puppycrawl/tools/checkstyle/BaseCheckTestSupport.java

```diff
@@ -177,6 +176,53 @@ protected static void verifyAst(String expectedTextPrintFileName, String actualJ
     }
 
     /**
+     * Verifies the Java and Javadoc Ast for the supplied java file against the supplied text file.
+     * @param expectedTextPrintFileName expected text ast tree representation.
+     * @param actualJavaFileName actual text ast tree representation.
+     * @throws Exception if exception occurs during verification.
+     */
+    protected static void verifyJavaAndJavadocAst(String expectedTextPrintFileName,
```

Comment:
This one shall be needed in #4451. I didn't realize we won't be needing it here. I shall take it out.